content/index.mdx

```diff
@@ -1,19 +1,20 @@
 ---
-template: "base"
+template: 'base'
 ---
-import Header from "components/organisms/SiteHeader"
-import Footer from "components/organisms/SiteFooter"
-import SubscribeSection from "components/molecules/SubscribeSection"
 
-import GitFlowIcon from "media/icons/gitflow.svg"
-import ReportsIcon from "media/icons/reports.svg"
-import BoxIcon from "media/icons/cube.svg"
+import Header from 'components/organisms/SiteHeader'
+import Footer from 'components/organisms/SiteFooter'
+import SubscribeSection from 'components/molecules/SubscribeSection'
```

Comment:
again, doesn't look like a linter/formatter

Reply:
You can reproduce it in GitHub Codespaces by creating a new "local" branch, modifying that file (from the `master` branch) and committing it through the user interface. It fails to run the hooks automatically, but then I run the command again manually and it "works" producing this.

![Captura de pantalla 2021-08-17 a las 21 46 40](https://user-images.githubusercontent.com/11387611/129790664-e16b12e8-cdf6-4bb0-bd7c-c00f1e64050d.png)
![Captura de pantalla 2021-08-17 a las 21 47 16](https://user-images.githubusercontent.com/11387611/129790674-a6ae93de-8733-438f-aa4c-9c727fb03955.png)

Reply:
different node version? For me, manually doing `npm up && yarn format-all && yarn pretty-quick` doesn't make these changes (`nodejs==14.17.4`).

Reply:
For me, manually running `npm up && yarn format-all && yarn pretty-quick` **does** make these changes. Using `nodejs==v14.15.1` with vanilla codespaces.